dist/installer/update_metadata.sh

```diff
@@ -35,7 +35,7 @@ then
   DEFAULT_TARGET_DIR="@homeDir@/LibrePCB"
 elif [[ "$TARGET_NAME" == mac* ]]
 then
-  DEFAULT_TARGET_DIR="@homeDir@/Applications/LibrePCB"
+  DEFAULT_TARGET_DIR="/Applications/LibrePCB"
```

Comment:
I just saw that there is also a [predefined variable](https://doc.qt.io/qtinstallerframework/scripting.html#predefined-variables) available for the applications directory:

```suggestion
  DEFAULT_TARGET_DIR="@ApplicationsDir@/LibrePCB"
```

Reply:
Ah, nice find!